classes/04class/exercises/c04-iac02/mail2kvanitha/main.tfvar

```diff
@@ -0,0 +1,5 @@
+vpc-cidr = "10.0.0.0/16" 
```

Comment:
One tip here if you would like to avoid typing all the subnets block is to use [`cidrsubnet`](https://www.terraform.io/docs/configuration/functions/cidrsubnet.html):

```
# for var.vpc-cidr => "10.0.0.0/16"
cidrsubnet(var.vpc-cidr, 8, 101) -> 10.0.101.0/24
cidrsubnet(var.vpc-cidr, 8, 102) -> 10.0.102.0/24
cidrsubnet(var.vpc-cidr, 8, 201) -> 10.0.201.0/24
cidrsubnet(var.vpc-cidr, 8, 202) -> 10.0.202.0/24
```